requirements-dev.txt

```diff
@@ -5,7 +5,7 @@ responses~=0.10
 flake8~=3.7
 black~=22.3
 Pillow~=9.0
-mkdocs~=1.1
+mkdocs~=1.3
 mkdocs-material~=7.3
 mkdocs-material-extensions~=1.0
-mkdocstrings[python]~=0.18
+mkdocstrings[python]~=0.19
```

Comment:
.github/workflows/verify-docs.yml lines 25-27 look like they should be removed because they'd be installed by line 20, right?

Reply:
Bump @gegnew.